README.md

```diff
@@ -82,15 +82,17 @@ After you're finished please remove all the comments and instructions!
 
 > screenshots of the project
 
-![Screenshot](./1.jpg)
-![Screenshot](./2.jpg)
-![Screenshot](./3.jpg)
-![Screenshot](./4.jpg)
-![Screenshot](./5.jpg)
-![Screenshot](./6.jpg)
-![Screenshot](./7.jpg)
-![Screenshot](./8.jpg)
-![Screenshot](./query multiple tables.jpg)
+![Screenshot](./assets/1.jpg)
+![Screenshot](./assets/2.jpg)
+![Screenshot](./assets/3.jpg)
+![Screenshot](./assets/4.jpg)
+![Screenshot](./assets/5.jpg)
+![Screenshot](./assets/6.jpg)
+![Screenshot](./assets/7.jpg)
+![Screenshot](./assets/8.jpg)
+![Screenshot](./assets/10.jpg)
```

Comment:
- [ ] Hey @Mekuseo, You have done well so far but I will like you to adjust your README file. The screenshots from your database should be uploaded on the PR description section not on the README file so as to fulfill the requirements for this task. kindly fix this

Reply:
files uploaded